dec.py

```
@@ -0,0 +1,95 @@
+# Method 2: zlib decompression (with header)
+print("\n2. Trying zlib decompression (with header)...")
+try:
+    decompressed = zlib.decompress(data)
+    print("✓ SUCCESS with zlib!")
+    print(f"Decompressed size: {len(decompressed)} bytes")
+    print("\nFirst 1000 characters:")
+    print("-" * 60)
+    try:
+        text = decompressed.decode("utf-8")
+        print(text[:1000])
+    except Exception:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)